data/config/toolbox.conf

```diff
@@ -0,0 +1,13 @@
+[general]
+# Default used distribution (e.g., fedora, rhel)
+# Taken from ID field in /etc/os-release
+## default_distro = "fedora"
+
+# Default version of a distribution (e.g., 34, 8.4)
+# Taken from VERSION_ID field in /etc/os-release
+## default_release = "34"
+
+# Default used image
+# Overrides 'default_distro' and 'default_release'
+# E.g., registry.fedoraproject.org/fedora-toolbox
+## default_image = "registry.fedoraproject.org/fedora-toolbox"
```

Comment:
The `:34` tag should probably be included. Otherwise it looks incomplete.

Reply:
Right